### ISSUE TITLE
Error: No module loaded!

### ISSUE THREAD

**Issue by jvlavl** [Nurrl/Duckuino#70]
Hello,

I have dowloaded duckuino to my local webserver and configured this virtual host to listen to port 81
When I open the site http://hostip:81 site is loading correctly.

pasting the ducky code in it the compile button lights up but when I hit it It gives me the error No module loaded!

How can I solve this?

__Input code:__
```vb
DELAY 3000
GUI r
DELAY 500
STRING notepad
DELAY 500
ENTER
DELAY 750
STRING Hello World!!!
ENTER
```

Rgds,

John

**Comment by Nurrl** [comment]
Hi @jvlavl, I gonna need a screenshot please.

**Comment by jvlavl** [comment]
Hello,

Here is the requested screenshot.
![modulemissing](https://user-images.githubusercontent.com/39025097/41215378-51aef968-6d50-11e8-9e41-4e1474038955.PNG)

Thanks in advance

**Comment by Nurrl** [comment]
Hi @jvlavl, have you selected a module with the caret on the compile button ?

**Comment by jvlavl** [comment]
Hello,

That"s also part of the issue i Can 't select a module. Aldo they are present in the root directory of the site.

Just die a git globe info/var/www/ducky apache config is pointing to /var/www/Lucky

Rgds,

John

**Comment by Nurrl** [comment]
Can you open the Javascript console and refresh + try to compile and send a screnshot of the console ?

**Comment by Nurrl** [comment]
I think there is a problem of module loading ^^

**Comment by jvlavl** [comment]
Hello,
Hereby the requested screenshots

![modulemissing](https://user-images.githubusercontent.com/39025097/41296988-6f123fa6-6e5e-11e8-81eb-30f4009cc5d8.PNG)

![modulemissing1](https://user-images.githubusercontent.com/39025097/41297001-73614b9c-6e5e-11e8-94c3-d9345b823c4b.PNG)

Rgds,

John

**Comment by Nurrl** [comment]
Hi @jvlavl, that's a non-issue, you can't access https://nurrl.github.io/LocaleKeyboard.js/locales/localeList with the client so the code breaks..
For now, Duckuino needs to access this file from internet to work properly.

**Comment by Plazmaz** [comment]
@Nurrl won't that drive us to CORS issues?

**Comment by Nurrl** [comment]
@Plazmaz CORS ?

**Comment by Nurrl** [comment]
@Plazmaz Yeah, I got it, yes it will, I'm trying to merge the two projects, but now I don't have much time to do it..
